transforms/angle-brackets/transform.js

```diff
@@ -149,6 +149,10 @@ function transformAttrs(tagName, attrs) {
     } else {
       _value = b.text(a.value.original || _EMPTY_STRING_);
     }
+
+    if (_value.chars && _value.chars.length > 0) {
+      _value.chars = JSON.stringify(_value.chars).slice(1, -1);
+    }
```

Comment:
I don't understand this. What are we stringifying something here?

Reply:
The reason i do this is because `_value` strips the `\` within the string:
![image](https://user-images.githubusercontent.com/18200519/71636724-00dbca80-2be9-11ea-879b-983b55bbbcfb.png)

hence: "Choose a "thing"..."
`JSON.stringify` is great at escaping strings. So I use it to properly escape strings and remove the first and last `double-quote` resulting in `"Choose a \"thing\"..."`

Reply:
isn't that a bug in the printer or template-recast?

Reply:
I didn’t consider that. I guess I could check how other parsers handle such things. I’ll look into this a little more.